src/main/java/com/uber/cadence/internal/sync/WorkflowInternal.java

```diff
@@ -228,26 +220,6 @@ public static <T> T newContinueAsNewStub(
             new ContinueAsNewWorkflowInvocationHandler(options, getWorkflowInterceptor()));
   }
 
-  /**
-   * Execute activity by name.
-   *
-   * @param name name of the activity
-   * @param resultClass activity return type
-   * @param args list of activity arguments
-   * @param <R> activity return type
-   * @return activity result
-   */
-  public static <R> R executeActivity(
```

Comment:
I wonder why is this removed?

Reply:
I'm not really sure why this exists, it's long unused and untested. I think it's best we just remove it.